.circleci/config.yml

```diff
@@ -311,7 +310,14 @@ jobs:
          pr-name: << pipeline.parameters.pr_number >>
       - pr_comment:
          when: on_success
-         comment: "Deployed prnet successfully"
+         comment: "\
+           <p>PR NET pr-<< pipeline.parameters.pr_number >> Successfully deployed!</p>\ 
+           <p>explorer=https://explorer-pr-<< pipeline.parameters.pr_number >> </p> \
+           <p>RPC=rpc-pr-<< pipeline.parameters.pr_number >>.prnet.taraxa.io </p>\
```

Comment:
The RPC is also an ingress and has HTTPS on front
but I do not know if the ledger team wants this var with HTTPS on front, please check with them

Reply:
/deploy